Samples/Tutorial Samples/YOLOv4ObjectDetection/YOLOv4ObjectDetection/YOLOv4ObjectDetection.csproj

```diff
@@ -151,7 +153,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
-      <Version>6.2.10</Version>
```

Comment:
removing this results in the warning here: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/NU1604
"Project dependency 'PackageA' (<= 9.0.0) does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results."

Reply:
Hmmm, you are right, I changed it back to 6.2.10 and mysteriously it is working for me now.  Weird.  Perhaps I was missing a solution restore before...